lib/MusicLibraryController.rb

```diff
@@ -0,0 +1,95 @@
+require 'pry'
+
+extend Concerns::Findable
```

Comment:
Module should `only` ever be `include`d or `extend`ed inside the body of a Class definition. This makes your Module globally accessible. Can you move it into the classes that specifically need it?